src/zc/table/fieldcolumn.py

```diff
@@ -160,9 +164,10 @@ def renderCell(self, item, formatter):
         return self.getRenderWidget(
             item, formatter, ignore_request)()
 
+
 class SubmitColumn(BaseColumn):
 
-    ###### subclass helper API (not expected to be overridden) ######
+    # subclass helper API (not expected to be overridden) ######
```

Comment:
If you're removing the `#####` decoration from the front of the comment, remove it from the back as we..